tests/suites/test_suite_ssl.data

```diff
@@ -3543,11 +3537,11 @@ Sanity test cid functions
 cid_sanity:
 
 Raw key agreement: nominal
-depends_on:MBEDTLS_SHA256_C
+depends_on:MBEDTLS_HAS_ALG_SHA_256_VIA_MD_OR_PSA_BASED_ON_USE_PSA
```

Comment:
Minor: that's correct, but that's not what the commit message says. Ideally this would have been too commits but splitting now is probably more trouble than it's wroth. However, you might want to edit the commit message if you're rewriting history at some point.